src/Mapster.Core/Utils/Extensions.cs

```diff
@@ -1,9 +1,12 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Linq;
 using System.Linq.Expressions;
+using System.Reflection;
 
 namespace Mapster.Utils
 {
-    static class Extensions
+    public static class Extensions
```

Comment:
Made this public to use from Mapster and Mapster.Tool. Maybe not ideal if it gets published?

Reply:
Looks fine to me to be honest, nothing problematic being exposed in that class.